proxy/httpp/interface.go

```diff
@@ -39,6 +41,19 @@ const (
 	XForwardedForNone = "none"
 )
 
+type XWebauthTreatment string
+
+const (
+	// Default behavior: set no additional headers on the forwarded request.
+	XWebauthNone XWebauthTreatment = "none"
+	// Sets:
+	//   * X-Webauth-Userid to the user's unique ID
+	//   * X-Webauth-Username to the user's username
+	//   * X-Webauth-Organization to the user's organization
+	//   * X-Webauth-Globalname to the user's "global name" (see oauth.(*Identity).GlobalName)
+	XWebauthSet = "set"
```

Comment:
did u mean to have this set as an untyped string instead of `XWebauthTreatment`?

Reply:
It's still typed - only the first constant in a block of constants need to specify type: https://www.sohamkamani.com/golang/enums/#enum-types

Reply:
Actually, I'm wrong, and the blog post is wrong, and the code above is wrong: https://go.dev/play/p/YaJgdoD7jgR

Fixed here and above

Reply:
(good catch, btw, I suspect I had started with a iota here and then changed)